test-network-k8s/scripts/kind.sh

```diff
@@ -115,6 +135,9 @@ function kind_init() {
   kind_create
   apply_nginx_ingress
   launch_docker_registry
+
+  pull_docker_images
+  # load_kind_image_plane
```

Comment:
worth removing this; or change to a comment saying that the docker images are pushed to a local registry rather than loaded into KIND.

Reply:
@mbwhite, how about below?
```
+ pull_docker_images
+ push_images_to_local
```